src/view/com/profile/ProfileSubpageHeader.tsx

```diff
@@ -72,54 +70,54 @@ export function ProfileSubpageHeader({
   }, [openLightbox, avatar])
 
   return (
-    <CenteredView style={pal.view}>
+    <CenteredView style={{ backgroundColor: tokens.color.l0 }}>
       {isMobile && (
-        <View
-          style={[
-            {
-              flexDirection: 'row',
-              alignItems: 'center',
-              borderBottomWidth: 1,
-              paddingTop: isNative ? 0 : 8,
-              paddingBottom: 8,
-              paddingHorizontal: isMobile ? 12 : 14,
-            },
-            pal.border,
-          ]}>
+        <Box
+          row aic
+          pt={web('s')}
+          pb='s'
+          px='m'
+          borderColor='l3'
+          borderBottomWidth={1}
+          gtMobile={{
+            px: 14
+          }}>
```

Comment:
Some of the styles in here could probably be simplified when we take a more holistic approach to components